### ISSUE TITLE
Filter on nested properties

### ISSUE THREAD

**Issue by VarunSaiTeja** [VarunSaiTeja/FirestoreLINQ#2]
Add filter on nested objects.

```csharp
Students.Where(x=>x.Address.Country=="India");
```

should be translated to

```csharp
students.WhereEqualTo("Address.Country","India")
```

**Comment by VarunSaiTeja** [comment]
The work is in progress on this enhancement. Will be available for next release

**Comment by VarunSaiTeja** [comment]
The enhancement has been released in v1.0.1.
